Karma/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -79,6 +79,7 @@
             }
         </section>
     </div>*@
+        </div>
```

Comment:
Seems strange for `</div>` to appear without the starting tag `<div>`. Is it a bug fix?